test/dataframe.jl

```diff
@@ -698,7 +698,7 @@ end
         describe(df, :detailed)
 
     # Test that it works on a custom function
-    describe_output.test_std = describe_output.std
+    describe_output.test_std=describe_output.std
```

Comment:
we need space here - this is not a keyword argument of a function:
```suggestion
    describe_output.test_std = describe_output.std
```